index.js

```diff
@@ -8,6 +8,7 @@ var Omise = require('42-cent-omise').factory;
 var Stripe = require('42-cent-stripe').factory;
 var Braintree = require('42-cent-braintree').factory;
 var PayPal = require('42-cent-paypal').factory;
+var Payeezy = require('42-cent-payeezy').factory
```

Comment:
You need to create a new repository, write the code for the module, then publish the module on npm. The important part here is `write the code`. Check how the other payment gateways are implemented please.